addons/info.go

```diff
@@ -0,0 +1,50 @@
+package addons
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/olekukonko/tablewriter"
+	"gopkg.in/errgo.v1"
+
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/cli/db"
+)
+
+func Info(app, addon string) error {
+	c, err := config.ScalingoClient()
+	if err != nil {
+		return errgo.Notef(err, "fail to get Scalingo client")
+	}
+	addonInfo, err := c.AddonShow(app, addon)
+	if err != nil {
+		return errgo.Mask(err, errgo.Any)
+	}
+
+	dbInfo, err := db.Show(app,addon)
+	if err != nil {
+		return errgo.Mask(err, errgo.Any)
```

Comment:
`errgo.Mask` is a deprecated way of handling the errors. Please use `errgo.Notef` (https://www.notion.so/scalingooriginal/Go-Style-Guide-de47c3b8d6c24c2e9b70c7b9c2f36660#158c056247cf463c9ae078893bf360b9)